contracts/MintAndBurnAuthorityV3.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.4.24;
+
+import "./DSAuth.sol";
+
+contract MintAndBurnAuthority is DSAuth {
+
+    mapping (address => bool) public allowList;
+
+    constructor(address[] _allowlists) public {
+        for (uint i = 0; i < _allowlists.length; i ++) {
+            allowList[_allowlists[i]] = true;
+        }
+    }
+
+    function canCall(
+        address _src, address _dst, bytes4 _sig
+    ) public view returns (bool) {
+        return ( allowList[_src] && _sig == bytes4(keccak256("mint(address,uint256)")) ) ||
+        ( allowList[_src] && _sig == bytes4(keccak256("burn(address,uint256)")) );
+    }
+
+    function addAllowList(address allowAddress) public onlyOwner{
```

Comment:
rename to ```addAllowAdress``` and ```removeAllowAddress```

Reply:
Have renamed the contract , used it as an extended contract or testnet